doc/conf.py

```diff
@@ -32,7 +32,7 @@
 # -- Project information -----------------------------------------------------
 
 project = 'The Plutus Platform and Marlowe'
-copyright = '2020, IOHK'
+copyright = '2021, IOHK'
```

Comment:
... or "2020, 2021, IOHK"? 

Also, what are we called these days?

Reply:
I have no idea. This seems fine.